components/ContributionForm.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <Button
+    v-if="btnWhite"
```

Comment:
the `v-if` is not needed here you could simply use `v-bind="$attrs"` https://vuejs.org/v2/api/#vm-attrs
```html
<Button
    v-bind="$attrs"
    class="typeform-share button"
    href="https://mesgfoundation.typeform.com/to/bOJ0qz"
    data-mode="popup"
    data-submit-close-delay="0"
  >Add contribution</Button>
```